src/scss/components/radio/mixins/_hxRadioControl.scss

```diff
@@ -0,0 +1,80 @@
+// ===== Mixins for <hx-radio-control> states =====
+
+// ========== PRISTINE ==========
+@mixin __radioControl($pseudo-state: null) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow;
+  }
+  @else if $pseudo-state == hover {
+    color: $cyan-500;
+
+    &::-ms-check {
+      background-color: $cyan-50;
+      color: $cyan-500;
+    }
+  }
+  @else {
+    color: $gray-500;
+
+    &::-ms-check {
+      background-color: $gray-0;
+      border: 1px solid currentColor;
+      color: inherit;
+    }
+  }
+}
+
+// ========== INVALID ==========
+@mixin __radioControl--invalid($pseudo-state: null) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow-invalid;
+  }
+  @else if $pseudo-state == hover {
+    background-color: $red-100;
+    // border: 2px solid currentColor;
```

Comment:
Please uncomment line 34, the border needs to be applied to retain currently generated CSS styles.

Reply:
Changes have been done, please review.